content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md

```diff
@@ -38,6 +38,14 @@ To generate service-level metrics directly in the Envoy proxies, follow these st
     $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/metadata-exchange_filter.yaml
     {{< /text >}}
 
+    or
+
+    Enable metadata exchange filter with WebAssembly sandbox using the following command:
+
+    {{< text bash >}}
+    $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/metadata-exchange_with_wasm_filter.yaml
```

Comment:
This file won't be available in `master` branch of istio/proxy, will it?

Reply:
no it won't

Reply:
Updated the link

Reply:
@PiotrSikora : PR is ready for review!

Reply:
Looks good, but we cannot merge it before https://github.com/istio/proxy/pull/2433.